src/source/PeerConnection/Rtcp.c

```diff
@@ -9,6 +9,7 @@ STATUS onRtcpPacket(PKvsPeerConnection pKvsPeerConnection, PBYTE pBuff, INT32 pB
 
     CHK(pKvsPeerConnection != NULL && pBuff != NULL, STATUS_NULL_ARG);
 
+    //MEMSET(&rtcpPacket, 0x00, SIZEOF(rtcpPacket));
```

Comment:
Mind uncommenting this?

Reply:
Enabling MSAN seems more time consuming than I thought, uncommenting it out and will follow up MSAN task in separate PR.